templates/pages/environment.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "Environment",
```

Existing page names are pluralized, e.g. Executions

```suggestion
  "name": "Environments",
```